docs/dev/developer-guides/building-on-zksync/events.md

```diff
@@ -14,30 +14,50 @@ At zkSync, events behave the same way as in Ethereum.
 ## Events filtering
 
 Filtering is used to query indexed data and provide lower-cost data storage when the data is not required to be accessed on-chain.
+While filtering, we advice to load events by block ranges (0-1999, 2000-3999, ...) and index the result on your end.
 
 These can be used in conjunction with the [Provider Events API](https://docs.ethers.io/v5/api/providers/provider/#Provider--event-methods) and with the [Contract Events API](https://docs.ethers.io/v5/api/contract/contract/#Contract--events).
 
+## Getting the events
 
+To get the events, use the following template:
 
-```solidity
-    abi = [
-  "event Transfer(address indexed src, address indexed dst, uint val)"
-];
+```js
+const ether = require("ethers");
+const contractABI = require("./ABI_JSON")
 
-contract = new Contract(tokenAddress, abi, provider);
+const getEvent = async () => {
+const usdtAdress = "<YOUR ADDRESS>"
```

Comment:
This code block looks weird... mentions YOUR ADDRESS for a usdt contract for example, from and to is the same block number, has an "event" parameter that doesn't say what this variable actually is, mentions PAST_EVENT that is nowhere in the code.

Needs some work in order to be a good code example

Reply:
Did a rework on the code sample. @githubdoramon